### ISSUE TITLE
biblatex @periodical entries are ignored

### ISSUE THREAD

**Issue by jboynyc** [jgm/pandoc-citeproc#238]
As I understand it, `@periodical` entries should be used for special issues and the like. Unfortunately that doesn't seem to work with `pandoc-citeproc`. Those entries are simply ignored.

    $ cat periodical.bib
    @periodical{AnnalsBigData,
        editor = {Dhavan V. Shah and Joseph N. Cappella and W. Russell Neuman},
        issuetitle = {Toward Computational Social Science},
        issuesubtitle = {Big Data in Digital Environments},
        journal = {Annals of the American Academy of Political and Social Science},
        volume = {659},
        year = {2015},
    }
    $ pandoc-citeproc --bib2yaml periodical.bib
    ---
    references:
    ...
    $

I'm assuming this isn't the intended behavior, but if it is, how elese should these entries be handled?

**Comment by nickbart1980** [comment]
Confirmed. 

pandoc-citeproc tries to follow the biblatex specs as closely as possible, and the biblatex manual says, “The title of the periodical is given in the title field. If the issue has its own title in addition to the main title of the periodical, it goes in the issuetitle field” – so using `title = {Annals of the American Academy of Political and Social Science},` gives the expected output.

Since there is no CSL item type that corresponds to `@periodical`, `article-journal` is used, but the output looks ok in most styles I tried. Feel free to lobby for a CSL `periodical` item type on their forums.

@jgm: *Some*, however minimal error message would of course be great in cases like this.
